src/elements/content-sidebar/activity-feed/task-new/TaskCompletionRuleIcon.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { TASK_COMPLETION_RULE_ANY } from '../../../../constants';
+import messages from './messages';
+import Tooltip from '../../../../components/tooltip';
+import IconAnyTask from '../../../../icons/general/IconAnyTask';
+import type { TaskCompletionRule } from '../../../../common/types/tasks';
+
+import './TaskCompletionRuleIcon.scss';
+
+type Props = {|
+    completionRule: ?TaskCompletionRule,
+|};
+
+const TaskCompletionRuleIcon = ({ completionRule }: Props): React.Node => (
+    <React.Fragment>
+        {completionRule === TASK_COMPLETION_RULE_ANY ? (
+            <Tooltip position="top-center" text={<FormattedMessage {...messages.taskAnyAffordanceTooltip} />}>
+                <span>
```

Comment:
Is this span needed?

Reply:
Not needed updating.

Reply:
Actually @jstoffan looks like the span is needed...not sure why exactly. But the tooltip fails to render with only an SVG as it's child. The span is used by the tether component to render the tooltip and apply all the tooltip classes. 

All usages of tooltips in BUIE have a div or span as a child. `InfoIconWithTooltip` as an example.

Reply:
The SVG might need to forward refs or spread props

Reply:
I think it's because of this line in Tooltip.js 

`        const bodyEl = bodyElement instanceof HTMLElement ? bodyElement : document.body;`

SVG's aren't instances of HTMLElement